CDP4Composition/CommonView/ViewModels/BehaviorDialogViewModel.cs

```diff
@@ -152,7 +152,7 @@ private IBehavioralModelKindViewModel CreateBehavioralModelKindViewModel(Behavio
                 case BehavioralModelKind.Other:
                     {
                         var elementDefinition = this.Container as ElementDefinition;
-                        var vm = new ScriptKindViewModel(this.Thing.BehavioralParameter, this.Thing.Script, this.Session, this, elementDefinition?.Parameter);
+                        var vm = new ScriptKindViewModel(this.Thing.BehavioralParameter.Select( t => t.Clone(false)), this.Thing.Script, this.Session, this, elementDefinition?.Parameter);
```

Comment:
This line is rather long. Please add line breaks for every parameter, which is what ReSharper would suggest.